### PR TITLE
Update translations with Travis and not during build

### LINKED ISSUE #1675: Do not update translation file during build
Use Travis to update the translations (the generated .pot is different on every computer/build). See scripts repo for how this can be done.

### PR DESCRIPTION
**Pull request by plata**
fixes #1675 

Note: I couldn't really test if the Travis part works.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -79,3 +88,12 @@ jobs:
       script:
         - bundle exec jekyll build
         - bundle exec htmlproofer ./_site --only-4xx --check-favicon --check-html --assume-extension
+
+env:
+  global:
+  # to create secure for travis-ci.com
```
Comment (madoar) [inline .travis.yml]: I don't really understand the comments.
Why repository should be cloned? (ours or a specific one from travis.org)
Where do we find the application `travis`?

File: i18n/push_translations.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+git config --global user.email "travis@travis-ci.org"
+git config --global user.name "Travis CI"
+
+head_ref=$(git rev-parse HEAD)
```
Comment (madoar) [inline i18n/push_translations.sh]: Some comments explaining the `if` cases would be nice.

File: .travis.yml (hunk 1 above)
Comment (plata) [reply .travis.yml]: Just the same comments as for scripts. Our repository must be cloned. https://github.com/travis-ci/travis.rb

File: i18n/push_translations.sh (hunk 2 above)
Comment (plata) [reply i18n/push_translations.sh]: I couldn't really add more than in the `echo` below (I only copied this from some tutorial).

**Pull request merged**